src/components/Entry.vue

```diff
@@ -201,7 +287,8 @@ export default {
       return (
         !this.isAnyUserIDPresent ||
         this.invalidInputMessage != "" ||
-        this.isCurrentEntryIncomplete
+        this.isCurrentEntryIncomplete ||
+        this.isBirthDateEntryIncomplete
```

Comment:
for later, maybe not now, but what would be good to have is a separate method which we can call, `areAllRequiredFieldsFilled` or something like that which would check if all required fields are filled up properly or not and then we can call that method (or computed var) here